testutil/test_helper.go

```diff
@@ -76,9 +77,11 @@ func NewGenericTestHelper(
 	helmReleaseName string,
 	externalIssuer,
 	uninstall bool,
+	httpClient http.Client,
 ) *TestHelper {
 	return &TestHelper{
```

Comment:
Looks like there is also `KubernetesHelper` variable in `TestHelper`. Maybe we can use that somewhere in the code using the returned value from `NewKubernetesHelper()`?

Reply:
Ah, nice catch - was only considering setting the private fields through this method, but it certainly makes sense to initialize the `KubernetesHelper` field as well. That would mean that `NewGenericTestHelper` would now also have to accept `k8sContext` as a parameter and also return the error returned from `NewKubernetesHelper`.

I have pushed the changes incorporating the above. PTAL! :smile: